cmd/age-keygen/keygen.go

```diff
@@ -103,7 +103,7 @@ func main() {
 	}
 
 	if fi, err := out.Stat(); err == nil {
-		if fi.Mode().IsRegular() && fi.Mode().Perm()&0004 != 0 {
+		if !convertFlag && fi.Mode().IsRegular() && fi.Mode().Perm()&0004 != 0 {
```

Comment:
Let's make this `if` more compact and move it to the `else` at the end of the function, before `generate(out)`.

```
if fi, err := out.Stat(); err == nil && fi.Mode().IsRegular() && fi.Mode().Perm()&0004 != 0 {
```

Reply:
Done, I think.